src/managers/KeybindManager.cpp

```diff
@@ -1980,12 +1972,29 @@ void CKeybindManager::bringActiveToTop(std::string args) {
     }
 }
 
-void CKeybindManager::pushActiveToBottom(std::string args) {
-    if (g_pCompositor->m_pLastWindow && g_pCompositor->m_pLastWindow->m_bIsFloating) {
-        g_pCompositor->changeWindowZOrder(g_pCompositor->m_pLastWindow, 0);
-        // if follow_mouse && mouse_refocus ?
-        g_pInputManager->simulateMouseMovement();
+void CKeybindManager::alterZOrder(std::string args) {
+    const auto WINDOWREGEX = args.substr(args.find_first_of(',') + 1);
+    const auto POSITION    = args.substr(0, args.find_first_of(','));
+    auto       PWINDOW     = g_pCompositor->getWindowByRegex(WINDOWREGEX);
+
+    if (!PWINDOW && g_pCompositor->m_pLastWindow && g_pCompositor->m_pLastWindow->m_bIsFloating)
```

Comment:
this will fail if provided with a window, and crash if m_pLastWindow is null.

```cpp
if (!PWINDOW && g_pCompositor->m_pLastWindow && g_pCompositor->m_pLastWindow->m_bIsFloating)
    PWINDOW = g_pCompositor->m_pLastWindow;

if (!PWINDOW) {
    Debug::log(ERR, "alterZOrder: no window");
    return;
}
```